### ISSUE TITLE
Package needs an update to be compatible with the latest nova API

### ISSUE THREAD

**Issue by kfirba** [antonioribeiro/nova-boolean-datetime-field#6]
Hey.

I have an `email_verified_at` field which I want to display as "Is Email Verified" and it seems like your package is exactly what I'm looking for. I tried using it but got the following error:

```
Declaration of PragmaRX\BooleanDatetime\BooleanDatetime::nullable() should be compatible with Laravel\Nova\Fields\Field::nullable($nullable = true, $values = NULL)
```

It seems like there has been some time since you last updated the package. Is it because it is abandoned? perhaps there is a "built-in" solution for this type of fields and the package is no longer needed?

Thanks!

**Comment by tekreme73** [comment]
Yes, you may be able to find the fix in the master branch. A pull request has been merged and close, but @antonioribeiro didn't make a release with it.

Related to PR #3
